crates/ironclaw_gateway/static/theme.css

```
@@ -1,34 +1,200 @@
 /* Shared IronClaw theme tokens */
 
+/* Brand type system — Geist (shared with ironclaw.com), served from
+   /fonts/*. Variable woff2 from the official `geist` npm package; Geist
+   Pixel Square is the stylistic face for uppercase tags/labels. */
+@font-face {
+  font-family: 'Geist';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)